Index: qtcreator/src/plugins/plugins.pro
===================================================================
--- qtcreator.orig/src/plugins/plugins.pro
+++ qtcreator/src/plugins/plugins.pro
@@ -66,10 +66,10 @@ isEmpty(DO_NOT_BUILD_QMLDESIGNER) {
 }
 
 
-isEmpty(QBS_INSTALL_DIR): QBS_INSTALL_DIR = $$(QBS_INSTALL_DIR)
-exists(../shared/qbs/qbs.pro)|!isEmpty(QBS_INSTALL_DIR): \
-    SUBDIRS += \
-        qbsprojectmanager
+#isEmpty(QBS_INSTALL_DIR): QBS_INSTALL_DIR = $$(QBS_INSTALL_DIR)
+#exists(../shared/qbs/qbs.pro)|!isEmpty(QBS_INSTALL_DIR): \
+#    SUBDIRS += \
+#        qbsprojectmanager
 
 # prefer qmake variable set on command line over env var
 isEmpty(LLVM_INSTALL_DIR):LLVM_INSTALL_DIR=$$(LLVM_INSTALL_DIR)
