Index: qtcreator/src/plugins/clangcodemodel/clangcodemodel.pro
===================================================================
--- qtcreator.orig/src/plugins/clangcodemodel/clangcodemodel.pro	2015-09-27 14:54:16.570761417 -0500
+++ qtcreator/src/plugins/clangcodemodel/clangcodemodel.pro	2015-09-27 14:55:51.205735495 -0500
@@ -9,7 +9,9 @@
 DEFINES += CLANG_VERSION=\\\"$${LLVM_VERSION}\\\"
 DEFINES += "\"CLANG_RESOURCE_DIR=\\\"$${LLVM_LIBDIR}/clang/$${LLVM_VERSION}/include\\\"\""
 
-unix:QMAKE_LFLAGS += -Wl,-rpath,\'$$LLVM_LIBDIR\'
+!llvm_disable_rpath {
+    unix:QMAKE_LFLAGS += -Wl,-rpath,\'$$LLVM_LIBDIR\'
+}
 
 SOURCES += \
     activationsequencecontextprocessor.cpp \
Index: qtcreator/src/tools/clangbackend/clangbackend.pro
===================================================================
--- qtcreator.orig/src/tools/clangbackend/clangbackend.pro	2015-08-18 05:02:51.000000000 -0500
+++ qtcreator/src/tools/clangbackend/clangbackend.pro	2015-09-27 14:57:41.536547126 -0500
@@ -16,5 +16,7 @@
 
 unix {
     !osx: QMAKE_LFLAGS += -Wl,-z,origin
-    QMAKE_LFLAGS += -Wl,-rpath,$$shell_quote($${LLVM_LIBDIR})
+    !llvm_disable_rpath {
+        QMAKE_LFLAGS += -Wl,-rpath,$$shell_quote($${LLVM_LIBDIR})
+    }
 }
