Index: qtcreator-3.4.0+dfsg~b1/src/plugins/coreplugin/icore.cpp
===================================================================
--- qtcreator-3.4.0+dfsg~b1.orig/src/plugins/coreplugin/icore.cpp	2015-03-07 17:33:01.497454249 -0600
+++ qtcreator-3.4.0+dfsg~b1/src/plugins/coreplugin/icore.cpp	2015-03-07 17:33:01.493454250 -0600
@@ -411,7 +411,7 @@
 QString ICore::documentationPath()
 {
     const QString docPath = QLatin1String(Utils::HostOsInfo::isMacHost()
-                                            ? "/../Resources/doc" : "/../share/doc/qtcreator");
+                                            ? "/../Resources/doc" : "/../share/qtcreator/doc");
     return QDir::cleanPath(QCoreApplication::applicationDirPath() + docPath);
 }
 
