Index: qtcreator/src/plugins/coreplugin/icore.cpp
===================================================================
--- qtcreator.orig/src/plugins/coreplugin/icore.cpp	2015-08-19 17:12:58.478729450 -0500
+++ qtcreator/src/plugins/coreplugin/icore.cpp	2015-08-19 17:12:58.474729501 -0500
@@ -426,7 +426,7 @@
 QString ICore::documentationPath()
 {
     const QString docPath = QLatin1String(Utils::HostOsInfo::isMacHost()
-                                            ? "/../Resources/doc" : "/../share/doc/qtcreator");
+                                            ? "/../Resources/doc" : "/../share/qtcreator/doc");
     return QDir::cleanPath(QCoreApplication::applicationDirPath() + docPath);
 }
 
