---
 src/plugins/coreplugin/icore.cpp |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/src/plugins/coreplugin/icore.cpp
+++ b/src/plugins/coreplugin/icore.cpp
@@ -425,7 +425,7 @@ QString ICore::userResourcePath()
 QString ICore::documentationPath()
 {
     const QString docPath = QLatin1String(Utils::HostOsInfo::isMacHost()
-                                            ? "/../Resources/doc" : "/../share/doc/qtcreator");
+                                            ? "/../Resources/doc" : "/../share/qtcreator/doc");
     return QDir::cleanPath(QCoreApplication::applicationDirPath() + docPath);
 }
 
